src/components/paragraphs/sum-calculator/sum-calculator.tsx

```diff
@@ -245,7 +245,7 @@ const SumCalculatorParagraph = ({
       <div className="sticky bottom-0 mt-72">
         <div
           ref={summaryRef}
-          className="relative left-1/2 max-h-dvh w-screen -translate-x-1/2 overflow-x-hidden overflow-y-scroll border-t-2 bg-fog-light"
+          className="relative left-1/2 max-h-dvh w-screen -translate-x-1/2 overflow-hidden border-t-2 bg-fog-light"
```

Comment:
We need the overflow scroll because on mobile devices the overlay can be taller than the device screen. This prevents them from seeing all contents in the details. That's an accessible issue.